tests/end2end/features/tabs.feature

```diff
@@ -1741,3 +1741,22 @@ Feature: Tab management
         And I run :undo
         And I run :message-info "Still alive!"
         Then the message "Still alive!" should be shown
+
+    Scenario: Passthrough mode override
+        When I run :set -u 1.txt input.mode_override 'passthrough'
```

Comment:
I suspect this URL pattern is wrong. We interpret `-u plain.string` as `*://plain.string/*` which `file:///somepath/data/numbers/1.txt` doesn't match (I assume it is opening it as an absolute path.
Try `file:///*/1.txt`

Reply:
Yep, after some debugging I also realised this, now working in 15f93546fe58748144c7ad6131c2875be5d140e1 :)